src/behavior-considered-undefined.md

```diff
@@ -58,7 +58,10 @@ Please read the [Rustonomicon] before writing unsafe code.
 * Invoking undefined behavior via compiler intrinsics.
 * Executing code compiled with platform features that the current platform
   does not support (see [`target_feature`]), *except* if the platform explicitly documents this to be safe.
-* Calling a function with the wrong call ABI or unwinding from a function with the wrong unwind ABI.
+* Calling a function with the wrong [call ABI][abi].
+* Deallocating a Rust stack frame without executing destructors
+  for local variables owned by the stack frame. This can occur
+  with C functions like `longjmp`.
```

Comment:
Isn't this also "Violating assumptions of the Rust runtime"?

Reply:
I'm not sure where this belongs, now, because IMO it _definitely_ doesn't belong in `panic.md`.

How about modifying the "Violating assumptions" bullet to have two sub-bullets?

> * Violating assumptions of the Rust runtime. This is only possible using mechanisms outside Rust. Most assumptions of the Rust runtime are currently not explicitly documented.
>   * For assumptions specifically related to unwinding, see [unwinding-ub].
>   * Deallocating a Rust stack frame without executing destructors for local variables owned by the stack frame is always undefined. This can occur with C functions like `longjmp`.

Reply:
> How about modifying the "Violating assumptions" bullet to have two sub-bullets?

Yes that works for me.